sample/F8/sessions

```diff
@@ -0,0 +1,1226 @@
+[
+   {
+      "id":1,
+      "short_abstract":null,
+      "long_abstract":null,
```

Comment:
No description for the session?
Please provide atleast one either short or long.